linux.cc

```diff
@@ -396,6 +397,22 @@ static long sys_getcpu(unsigned int *cpu, unsigned int *node, void *tcache)
     return 0;
 }
 
+#define __NR_sys_prlimit __NR_prlimit64
+static int sys_prlimit(pid_t pid, int resource, const struct rlimit *new_limit,
```

Comment:
Shouldn't you call this function prlimit() and publish it? Does the Linux system call differ from the glibc function?

Reply:
Will fix it in a new PR.